tests/testthat/helper.R

```diff
@@ -1,13 +1,3 @@
-#' Does code throw an rextendr_error?
-#'
-#' `expect_rextendr_error()` expects an error of class `rextendr_error`, as
-#' thrown by `ui_throw()`.
-#'
-#' @param ... arguments passed to [testthat::expect_error()]
-expect_rextendr_error <- function(...) {
-  testthat::expect_error(..., class = "rextendr_error")
-}
-
```

Comment:
since we are no longer using `ui_throw()` we do not inherit the `rextendr_error` for errors.